cartoframes/context.py

```diff
@@ -379,20 +379,10 @@ def delete(self, table_name):
             table_name (str): Name of table to delete
 
         Returns:
-            None
+            Boolean: True if table is removed
```

Comment:
`bool` instead of `Boolean`.

Values are usually in '`' marks.

We should also be specific that `False` indicates a failure to delete.

Reply:
What happens if a user tries to delete a table that doesn't exist? I think an error should be thrown.

Reply:
Yep, I think so too. Probably we should go with the first approach, returning True/False and so on, and adding the raise. I will also use the `_drop_table_query` method as Alberto suggested in https://github.com/CartoDB/cartoframes/pull/589#discussion_r272244178, but making it smarter

Reply:
I have done it. Waiting for the green light and then I will ping you again